packages/remix-react/fog-of-war.ts

```diff
@@ -193,10 +187,7 @@ export function useFogOFWarDiscovery(
       attributeFilter: ["data-discover", "href"],
     });
 
-    return () => {
-      fogOfWar?.controller?.abort();
```

Comment:
This is technically against the "rules of react" because we're not cleaning up our side effect, but functionally it shouldn't cause any issues since the requests are additive and shouldn't have race conditions